### ISSUE TITLE
5.45 does not launch at all

### ISSUE THREAD

**Issue by djo5296** [kangyu-california/PersistentWindows#261]
Coming from 5.43 and clicked on the prompt to upgrade to 5.45. I noticed the software did not launch since updating (no icon in taskbar, icon set to show in taskbar settings). Even on (multiple) restarts, software is not running. I tried running the auto start . bat file as admin and I get this result:

![image](https://github.com/kangyu-california/PersistentWindows/assets/30708437/564300a9-0d4b-4082-92b4-c41bd0d2c282)
The .ps1 file is definitely in the same folder as the .bat file btw

I downloaded 5.43 again and pasted zip contents over the same installation directory and software runs as normal, even on restart

**Comment by kangyu-california** [comment]
1. your picture shows you are running an older version of .bat file in 5.43, which would fail if the installation path contains space character. This has been fixed in 5.45.
2. If you have anti-virus software installed, some PW component file could be quarantined, please double check the integrity of unzipped files.

**Comment by djo5296** [comment]
I have reupgraded to 5.45 to double check and
1) it seems to run now, persisting across multiple computer restarts/shutdowns
2) the bat file runs correctly. albeit i created my own task scheduler since i originally had PW when it didnt have the bat file, and I almost thought it was cuz I disallowed unsigned powershell scripts from running (so it shows the execution policy error proving the bat file works, just not allowed). But I swear the error I had running the bat file had the same error message as 5.43. So it's possible that smth else entirely different happened during the self-update, which also seemed to affect the actual software launch.

regardless, it is running as it should. Should have tried to reupdate to confirm first. I appreciate the quick response and the continuous update of this software. cheers
